docs/deploy/vitis_ai.rst

```diff
@@ -285,8 +305,10 @@ platform. The following development boards can be used out-of-the-box:
 
 Edge hardware setup
 ^^^^^^^^^^^^^^^^^^^
++------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
I don't think you need this for the note block?

Reply:
If the note block works as expected, it should have a frame around it?
For example, the "Note" block in this page: https://tvm.apache.org/docs/tutorials/get_started/cross_compilation_and_rpc.html#sphx-glr-tutorials-get-started-cross-compilation-and-rpc-py

Reply:
I see, we have been looking at this file in the repo itself. We will remove the box.

Reply:
You can use `make docs` to build the docs locally so that you can check how it looks like in your browser. Or you can use online RST compiler such as https://livesphinx.herokuapp.com/ for a quick check, although the doc theme is different.